### DISCUSSION TITLE
Support H.265 HEVC for RTSP stream

### DISCUSSION THREAD

**Discussion by touzh** [OvenMediaLabs/OvenMediaEngine#1481] [Ideas] [answered]
Since OME has support H.265 but when using RTSP it's say 
`E [StreamCollector:69] RtspcStream | rtspc_stream.cpp:541  | h265stream - Unsupported codec  :`

This is my RTSP Body
```
 I [StreamCollector:69] RtspcStream | rtspc_stream.cpp:364  | SDP : v=0
o=- 1702557113438239 1702557113438239 IN IP4 10.100.90.2
s=Media Presentation
e=NONE
b=AS:5050
t=0 0
a=control:rtsp://10.100.90.2:554/Streaming/channels/101/
m=video 0 RTP/AVP 96
c=IN IP4 0.0.0.0
b=AS:5000
a=recvonly
a=x-dimensions:2560,1440
a=control:rtsp://10.100.90.2:554/Streaming/channels/101/trackID=1
a=rtpmap:96 H265/90000
a=fmtp:96 sprop-vps=QAEMAf//AWAAAAMAsAAAAwAAAwCZvAk=; sprop-sps=QgEBAWAAAAMAsAAAAwAAAwCZoAFAIAWhY2+SRS/NwEBAQCA=; sprop-pps=RAHA8saNA7JA
a=Media_header:MEDIAINFO=494D4B48010300000400050000000000000000000000000000000000000000000000000000000000;
a=appversion:1.0
```

Could you please make it support H.265?

**Comment by getroot** [comment, marked as answer]
OME does not yet support H.265 in RTSP. But this is in our backlog.

**Reply by farbhaus** [reply]
@getroot any update on HEVC support? I'd love to be able to stream and playback h265 422 10bit

**Reply by getroot** [reply]
I haven’t had the time to add HEVC to RTSP yet. This task has been pushed back due to higher-priority tasks.

Are you aware that OME supports HEVC on SRT?

**Reply by ehspill** [reply]
@getroot any update? most hq cams come with h265 streams by now :/

**Reply by farbhaus** [reply]
> I haven’t had the time to add HEVC to RTSP yet. This task has been pushed back due to higher-priority tasks.
> 
> 
> 
> Are you aware that OME supports HEVC on SRT?

Yes! I'm going to try it soon, but the question is which flavor of h265 it supports. Does it do 422 chroma subsampling and 10bit?
